extensions/industry/healthcare/fhir/fieldgroups/careplan.schema.json

```diff
@@ -0,0 +1,250 @@
+{
+   "meta:license":[
+      "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+      "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+      "you may not use this file except in compliance with the License. You may obtain a copy",
+      "of the License at https://creativecommons.org/licenses/by/4.0/"
+   ],
+   "$id":"https://ns.fhir.org/fieldgroup/careplan",
+   "$schema":"http://json-schema.org/draft-06/schema#",
+   "title":"Care Plan",
+   "type":"object",
+   "meta:tags":{
+      "industry":[
+         "healthcare_services"
+      ]
+   },
+   "meta:intendedToExtend":[
+      "https://ns.adobe.com/xdm/context/profile",
+      "https://ns.adobe.com/xdm/context/account"
```

Comment:
Is there a reson this is required in the b2b classes? `https://ns.adobe.com/xdm/context/account` is a b2b class